test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -7831,6 +7831,32 @@ def forward(self, x, window_length: int):
         output = module(x, win_length)
         self.run_test(module, (x, win_length))
 
+    @disableScriptTest()
+    @skipIfUnsupportedMinOpsetVersion(12)
+    def test_tensordot_dim_count(self):
+        class M(torch.nn.Module):
+            def forward(self, x, y):
+                output = torch.tensordot(x, y, 2)
+                return output
+
+        x = torch.randint(6, (7, 5, 3, 4))
+        y = torch.randint(6, (3, 4, 8, 9))
+
+        self.run_test(M(), (x, y))
```

Comment:
you can extend this test with dynamic input size (test with input of different sizes after export), by using argument `dynamic_axes` and `test_with_inputs`

Reply:
Add one more test for this.